src/UniversalTelegramBot.h

```diff
@@ -125,7 +129,7 @@ class UniversalTelegramBot {
   unsigned int waitForResponse = 1500;
   int _lastError;
   int last_sent_message_id = 0;
-  int maxMessageLength = 1500;
+  int maxMessageLength = 10500;
```

Comment:
I'm concerned about the impact this will have on memory, does it need to be so large?

Reply:
Ok, this is certainly a point of discussion and this is probably application dependent. I created a chat bot using the 1500 length setting and found that it did not work at all because messages were incomplete rendering the reply useless. In effect he bot got stuck doing nothing so that I thought my code was bad. Debugging into it gave me the hint to modify the message size.

I suggest making this an obvious parameter for the library by moving it to private and adding at least a set method to make clear it is meant for application management. In addition, it could become a parameter of the contructor if you don't mind.

If you want to go ahead with this change, I can offer to make the necessary modification.